jobs/move_extract.py

```diff
@@ -84,15 +86,6 @@ def sort():
         args = list(args)
         for key, val in new_args.items():
             args.append('%s=%s' % (key, val))
-        if not any(a.startswith('--provider=') for a in args):
```

Comment:
Unrelated.

I just keep refactoring this file each time I migrate another env over to a flag (first was --extract, hence the name). We no longer use KUBERNTES_PROVIDER in our .env files (and have a unit test to enforce this) so the --provider migration logic is no longer necessary